src/commands.ts

```diff
@@ -29,18 +29,20 @@ export class RuntimeCommandExecutor implements CommandExecutor {
     }
 
     async isMinimumScannerPluginInstalled(minVersion: string): Promise<boolean> {
-        const command = 'sf plugins inspect @salesforce/sfdx-scanner --json'
-        const cmdOut: CommandOutput = await this.dependencies.execCommand(command)
+        const pluginName = '@salesforce/sfdx-scanner'
+        const command = `sf plugins inspect ${pluginName} --json`
+        const runSilently = true
+        const cmdOut: CommandOutput = await this.dependencies.execCommand(command, {}, runSilently)
         if (cmdOut.exitCode !== 0) {
             return false
         }
         try {
-            const pluginMetadataArray: PluginMetadata[] = JSON.parse(cmdOut.stdout) as PluginMetadata[]
-            if (pluginMetadataArray.length !== 1) {
+            const pluginsFound: PluginMetadata[] = JSON.parse(cmdOut.stdout) as PluginMetadata[]
```

Comment:
Are there cases where this array would be of length greater than 1? If so, is the first entry still always going to be the one we want?